tests/TestCase.php

```diff
@@ -13,54 +11,37 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->setUpDatabase($this->app);
+        $this->loadMigrationsFrom(realpath(__DIR__.'/fixtures/migrations'));
     }
 
     protected function getPackageProviders($app)
     {
-        return [TranslatableServiceProvider::class];
+        return [
+            \Orchestra\Database\ConsoleServiceProvider::class,
+            TranslatableServiceProvider::class,
+        ];
     }
 
     /**
      * @param \Illuminate\Foundation\Application $app
      */
     protected function getEnvironmentSetUp($app)
     {
-        $this->initializeDirectory($this->getTempDirectory());
-        file_put_contents($this->getTempDirectory().'/.gitignore', '*'.PHP_EOL.'!.gitignore');
-
-        $app['config']->set('database.default', 'sqlite');
-        $app['config']->set('database.connections.sqlite', [
-            'driver'   => 'sqlite',
-            'database' => "{$this->getTempDirectory()}/database.sqlite",
-            'prefix'   => '',
-        ]);
+        $this->setDatabase($app['config']);
     }
 
     /**
-     * @param \Illuminate\Foundation\Application $app
+     * Set the database.
+     *
+     * @param \Illuminate\Contracts\Config\Repository $config
      */
-    protected function setUpDatabase($app)
+    protected function setDatabase($config)
     {
-        file_put_contents($this->getTempDirectory().'/database.sqlite', null);
-
-        $app['db']->connection()->getSchemaBuilder()->create('test_models', function (Blueprint $table) {
```

Comment:
I'd prefer the migrations to be in the `TestCase` instead of a separate file. Could you restore this?